### DISCUSSION TITLE
Performance on Raspi 4

### DISCUSSION THREAD

**Discussion by ghost** [fayaaz/mixxx-pi-gen#25] [General]
Do you have any performance specs for this build? 
As you know, Raspi 4 boards have different RAM-sizes, so I want to
buy a suitable board.

**Comment by slashtechno** [comment]
@MaxHausmann My experience when using Mixxx on my Raspberry Pi 4 (4 GB RAM) is it works nicely, however, recording quality is quite bad.

You could also install [Raspberry Pi Desktop](https://www.raspberrypi.org/software/raspberry-pi-desktop/) in a VM on your computer and try different RAM amounts.

**Comment by fayaaz** [comment]
For 2 decks it seems like the 2GB version is acceptable. YMMV depending on acceptable latencies and sampling rates of files you're using.

For recording @slashtechno the limiting factor would possibly be the sd card storage - not sure of a way around this. I actually use another raspberry pi mounted through samba for my library.

**Comment by ghost** [comment]
Thank you for your answers. I will get the 4 GB version.
I want to use it with two decks and a Pioneer DDJ-400 controller.

Sorry for opening an issue instead of a discussion.

**Reply by MadTiXx** [reply]
I can agree.
I also use pi4 with 4gb works good.

Greetings
